tasks/internal/filter-already-released-advisory-images-task/filter-already-released-advisory-images-task.yaml

```diff
@@ -138,6 +146,23 @@ spec:
 
           echo "Comparing against: $ADVISORY_FILE"
 
+          # Track this advisory as the latest (since they're sorted by modification time, newest first)
+          if [[ -z "$LATEST_ADVISORY_URL" ]]; then
+            # Extract advisory details for URL generation
+            ADVISORY_TYPE=$(yq -r '.spec.type' "$ADVISORY_FILE")
+            ADVISORY_NAME=$(yq -r '.metadata.name' "$ADVISORY_FILE")
+            
+            # Determine URL prefix based on repository
+            if [[ "${GIT_REPO}" == *"/rhtap-release/"* ]]; then
+              ADVISORY_URL_PREFIX="https://access.stage.redhat.com/errata"
+            else
+              ADVISORY_URL_PREFIX="https://access.redhat.com/errata"
+            fi
```

Comment:
We repeatedly determine the same info (same URL prefix) on each loop.
Perhaps, from a performance and logical structure point of view, it would be better to move URL prefix determination outside the for loop?